src/core/BundleSplit.ts

```diff
@@ -42,6 +42,7 @@ export class BundleSplit {
         config.plugins = [].concat(config.plugins || [])
         config.standalone = false;
         const fuse = FuseBox.init(config);
+        new Bundle(name, fuse, producer);
```

Comment:
How so?  According to this, the split bundles end up in the directory specified by `dest` within the split config.  Shouldn't the source map files end up in the same directory as the js files?

![image](https://cloud.githubusercontent.com/assets/5567815/25576414/9b56ce6a-2e13-11e7-940d-1a2b6c852d8e.png)